src/main/webapp/join-room.js

```diff
@@ -13,11 +13,46 @@ const createRoom = async (ev) => {
       },
     });
   
-    // Sends user to room
+    // Sends user to room.
     window.location.href = 'lobby.html?roomId=' + room.roomId;
 }
 
+const joinRoom = async (ev) => {
+  ev.preventDefault();
+
+  // Get url from user input.
+  let url = document.getElementById('room-url-id').value;
+
+  // Split url at '?' and saves second half (url parameter).
+  let paramString = url.split('?')[1]; 
+  let queryString = new URLSearchParams(paramString);
+
+  // Saves value into roomId (pair[0] would be key).
+  let roomId = "";
+  for (let pair of queryString.entries()) {
+	  roomId = pair[1]; 
+  } 
```

Comment:
Can you factor these lines into their own parseRoomId() function which takes a URL as input and returns the room ID? In addition to making the joinRoom function smaller and easier to read, we could also add tests for the parseRoomId() function, which I think you should also do in this PR.

Let me know if you need any help with this

Reply:
@pruess-google will do